packages/sources/ncfx/CHANGELOG.md

```diff
@@ -1,763 +1,31 @@
 # @chainlink/ncfx-adapter
 
-## 2.1.15
-
-### Patch Changes
```

Comment:
Was there a specific motivation for removing the CHANGELOG data?

Reply:
Ah good catch. It got overwritten by v3's changelog. I'll restore it to v2's

Reply:
Fixed in the latest commit